pdr_backend/sim/sim_engine.py

```diff
@@ -58,6 +61,24 @@ def __init__(
         else:
             self.multi_id = str(uuid.uuid4())
 
+        use_live_data = self.ppss.predictoor_ss.use_live_stake_and_accuracy
+
+        if use_live_data:
+            data = get_avg_stake_and_accuracy_for_feed(self.predict_feed)
+            if data:
+                self.others_accuracy, self.others_stake = data
+                self.others_stake = self.others_stake.amt_eth
```

Comment:
I think the `self.others_stake` is referring to total stake for an epoch, but the stake amount from the api( `self.others_stake` ) is per day, so for an accurate stake you have to divide that by number of epochs